cmd/runtimetest/main.go

```diff
@@ -166,7 +166,19 @@ func validateCapabilities(spec *rspec.Spec) error {
 	}
 
 	expectedCaps := make(map[string]bool)
-	for _, ec := range spec.Process.Capabilities {
+	for _, ec := range spec.Process.Capabilities.Bounding {
+		expectedCaps[ec] = true
+	}
+	for _, ec := range spec.Process.Capabilities.Effective {
+		expectedCaps[ec] = true
```

Comment:
This isn't right.  You'll need different maps for each `spec.Process.Capabilities.*` and then different checks for each map below.  For example, the current `actuallySet := processCaps.Get(capability.EFFECTIVE, cap)` should be compared against an expected value from `spec.Process.Capabilities.Effective`.